### ISSUE TITLE
Prism.Core.7.0.0.396 is causing issue in Outlook VSTO Add-in Project

### ISSUE THREAD

**Issue by m5112638** [PrismLibrary/Prism#1370]
### Description

Prism.Core.7.0.0.396 is causing issue in Outlook VSTO Add-in project during compilation, the error is "The 'FindRibbons' task failed unexpectedly".

### Steps to Reproduce

1. Launch Visual Studio 2017, and then create a "Outlook 2013 and 2016 VSTO Add-in" project
2. Compile the project - passed, no errors
3. Add Prism.Unity.7.0.0.336-pre NuGet package to the project (this will automatically include Prism.Wpf and Prism.Core), make sure Prism.Core.7.0.0.396 is used
4. Compile the project - passed, no errors
5. Add a new C# class file named Bootstraper.cs as shown below to the project
    class Bootstrapper : UnityBootstrapper
    {
           protected override Prism.Logging.ILoggerFacade CreateLogger()
           {
               return new Prism.Logging.TraceLogger();
           }
    }
6. Compile the project again. Error is shown in the Error List: "The 'FindRibbons' task failed unexpectedly"

### Expected Behavior
Should not cause any issues.

### Actual Behavior
Error: The "FindRibbons" task failed unexpectedly

### Basic Information

- Version with issue: **Prism.Core.7.0.0.362** and **Prism.Core.7.0.0.396**
- Last known good version: **Prism.Core.7.0.0.336-pre**
- Xamarin.Forms version: N/A
- IDE: Visual Studio 2017

### Screenshots

REQUIRED﻿### Reproduction Link
[OutlookAddIn1.zip](https://github.com/PrismLibrary/Prism/files/1735248/OutlookAddIn1.zip)

**Comment by brianlagunas** [comment]
If writing production apps, you should use the current v6.3 for WPF.  If you must use the preview, you must upgrade Unity first.  You might be better off using our CI builds if you are trying to use Prism for WPF v7.  It is still being worked on and is not stable yet.

**Comment by m5112638** [comment]
Thanks Brian. It is the official release Prism.Core.7.0.0.362&396 that is causing the issue. When I switch to use Prism.Core.7.0.0.336-pre, it compiles and works.

**Comment by brianlagunas** [comment]
You cannot update Prism.Core packages with WPF 6.3, they are not binary compatible.  You should only be using v6.3 for all packages in WPF.  The preview will work because it is binary compatible, but it is in preview and not guaranteed to be stable.

**Comment by m5112638** [comment]
As I said, when I use Prism.Unity.7.0.0.336-pre, Prism.Wpf.7.0.0.336-pre with Prism.Core.7.0.0.336-pre, everything compiles and works (yes I upgraded Unity). However when I saw the prod release of Prism.Core NuGet package, I did an update, and broke the build. I believe it is something in prod release version Prism.Core.7.0.0.362&396 that is interfering with the compilation.

Are you recommending that we wait until Prism.Unity and Prism.Wpf's "pre" label is gone before using Prism 7? Thanks.
